iDine/iDine/Features/Order/Views/OrderView.swift

```diff
@@ -22,18 +22,24 @@ struct OrderView: View {
                             
                             Text("$\(item.price)")
                         }
-                    }
+                    }.onDelete(perform: deleteItems)
```

Comment:
If it's possible, could you move the ``onDelete`` modifier to the next line, so we keep the project style of modifiers always in the next line after the curly bracket ``}``.

Reply:
Done!